batch7crud-roll1/src/main/webapp/resources/css/style.css

```diff
@@ -0,0 +1,236 @@
+body{
+    background-color: #E2E2E2;
+    font-family: Arial;
+}
+
+a{
+    text-decoration: none;
+}
+
+.company-name h2{
+    color: #FFFFFF;
+    font-size: 20px;
+    font-weight: bold;
+}
+
+.header-menu{
+    float: right;
+    padding-top: 10px;
+}
+
+.header-menu ul{
+
+}
+
+.header-menu ul li{
+}
+
+.header-menu ul li a{
+    display: block;
+    padding: 10px;
+    color: #FFF;
+    background-color: #357ebd;
+    border-radius: 4px;
+}
+
+.navigation ul{
+
+}
+
+.navigation ul li{
+    border-bottom: 1px solid #444444;
+}
+
+.navigation ul li a{
+    display: block;
+    padding: 20px 15px;
+    color: #FFF;
+}
+
+.navigation ul li:hover{
+    background-color: #000000;
+}
+
+.main-content h2{
+    font-size: 20px;
+    text-transform: uppercase;
+    color: #000;
+    font-weight: 800;
+    padding: 15px;
+    border-bottom: 2px solid #00CA79;
+}
+
+.employee-form{
+    padding: 15px;
+}
+
+.employee-table{
+    padding: 15px;
+}
+
+.table{
+    width: 100%;
+}
+
+.table, th, td{
+    border:1px solid #ddd;
+}
+
+.table th, td{
+    padding: 8px;
+}
+
+.table th{
+    font-weight: bold;
+    text-align: left;
+}
+
+.table > tbody > tr:hover > td{
+    background-color: #f5f5f5;
+}
+
+.form-group{
+    margin: 10px 0;
+    position: relative;
+}
+
+.form-group label{
+    display: inline-block;
+    max-width: 100%;
+    padding: 5px 0;
+    font-weight: bold;
+}
+
+.form-group span{
+    color: red;
+    position: absolute;
+    left: 300px;
+    top: 33px;
+}
+
+.form-control{
+    display: block;
+    width: 25%;
+    padding: 6px 12px;
+    font-size: 14px;
+    color: #555;
+    background-color: #fff;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    outline: none;
+}
+
+.button{
+    display: inline-block;
+    padding: 6px 12px;
+    margin-bottom: 0;
+    font-size: 14px;
+    cursor: pointer;
+    font-weight: normal;
+    text-align: center;
+    border: 1px solid transparent;
+    border-radius: 4px;
+}
+
+.button-primary{
+    color: #fff;
+    background-color: #357ebd;
+}
+
+.button-danger{
+    color: #fff;
+    background-color: #d9534f;
+    border-color: #d43f3a;
+}
+
+.login-content{
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Check this potential box model size issue [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=css%3Abox-model)